packages/jsii-pacmak/test/expected.jsii-calc-lib/java/src/main/java/software/amazon/jsii/tests/calculator/lib/MyFirstStruct.java

```diff
@@ -13,12 +13,14 @@
     @Deprecated
     @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Deprecated)
     java.lang.Number getAnumber();
+
     /**
      * A string value.
      */
     @Deprecated
     @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Deprecated)
     java.lang.String getAstring();
+
     /**
      */
```

Comment:
Remove comment blocks with no content?

Reply:
I'm going to record that as a future feature request/improvement. It's out of scope for this change.